src/res/translation/translation_fr_FR.ts

```diff
@@ -1218,7 +1214,7 @@
     </message>
     <message>
         <source>ASIO Setup</source>
```

Comment:
I think this doesn't exist anymore. It's now ASIO Device Setup

Reply:
Fixed.
Ready to be merged now.